tools/android/android_extensions.bzl

```diff
@@ -19,7 +19,7 @@ load("//tools/build_defs/repo:http.bzl", "http_archive", "http_jar")
 def _remote_android_tools_extensions_impl(_ctx):
     http_archive(
         name = "android_tools",
-        sha256 = "1afa4b7e13c82523c8b69e87f8d598c891ec7e2baa41d9e24e08becd723edb4d",  # do_not_remove_this_android_tools_update_marker
+        sha256 = "1afa4b7e13c82523c8b69e87f8d598c891ec7e2baa41d9e24e08becd723edb4d",  # DO_NOT_REMOVE_THIS_ANDROID_TOOLS_UPDATE_MARKER
         url = "https://mirror.bazel.build/bazel_android_tools/android_tools_pkg-0.27.0.tar.gz",
```

Comment:
Also https://github.com/bazelbuild/bazel/commit/65de62884378aa7fb6177d1f350eed7b7d90067e#diff-a9eccf018d9fcd9183819d240f5af2c97ba11266a26f045590521d1537219c0cL87 removed the `.gz`

Reply:
the script used `sed -i` so that's why it worked at all in the first place, but it's good to be consistent. we're migrating the pipeline that generates these artifacts and we'll fix the `.gz` part along with that.